package.json

```diff
@@ -14,6 +14,8 @@
   "dependencies": {
     "core-js": "^3.6.4",
     "vue": "^2.6.11",
+    "vue-async-computed": "^3.8.2",
+    "vue-async-computed-decorator": "0.0.4",
```

Comment:
We don't use any of these two dependencies, so we should delete them.

Reply:
Removed